pmm-admin_test.go

```diff
@@ -402,6 +403,40 @@ Get http://xyz/qan-api/ping: dial tcp: lookup xyz.*: no such host
 	assertRegexpLines(t, expected, string(output))
 }
 
+func testConfigServerHideCredentials(t *testing.T, data pmmAdminData) {
+	defer func() {
+		err := os.RemoveAll(data.rootDir)
+		assert.Nil(t, err)
+	}()
+
+	err := os.MkdirAll(data.rootDir+pmm.PMMBaseDir, 0777)
+	assert.Nil(t, err)
```

Comment:
Please use `assert.NoError()` in this and other cases – it checks more and generates better error messages.